src/pip/_internal/utils/misc.py

```diff
@@ -728,13 +782,15 @@ def read_text_file(filename):
     encodings = ['utf-8', locale.getpreferredencoding(False), 'latin1']
     for enc in encodings:
         try:
-            data = data.decode(enc)
+            # https://github.com/python/mypy/issues/1174
+            data = data.decode(enc)  # type: ignore
         except UnicodeDecodeError:
             continue
         break
 
     assert type(data) != bytes  # Latin1 should have worked.
```

Comment:
Change this to use `isinstance` and we should be able to remove the next `type: ignore`.